Libplanet.Explorer.Executable/Program.cs

```diff
@@ -60,8 +58,7 @@ public static async Task Main(string[] args)
         public void Schema()
         {
             var serviceCollection = new ServiceCollection();
-            serviceCollection.AddGraphQL()
-                .AddGraphTypes(typeof(LibplanetExplorerSchema<NullAction>));
+            serviceCollection.AddGraphQL(builder => builder.AddLibplanetExplorerSchema<NullAction>());
```

Comment:
It seems this line is tripping on 100 character limit?

Reply:
It was edited by my refactor without seeing.  Amended!